assets/test/adaptivity/scripting_test.ts

```diff
@@ -247,6 +248,31 @@ describe('Scripting Interface', () => {
       expect(valuey).toBe(varValueFormat2);
     });
 
+    it('should return the CSS as it is', () => {
```

Comment:
This is from an old PR that was already approved by @bsparks . By Mistake, i made these changes in Master branch and had created the PR on Master so closed that one and added this here.